docs/Model_Assumptions.rst

```diff
@@ -176,12 +176,117 @@ The confersion factors are derived from their `Gasoline Gallon Equivalents.<http
 
-- Energy consumption and feed-in at the same time
-- Bi-directional inverters
+Extension of KPIs
+#################
+:Limitation: Some important KPIs, usually required by developers, are currently not implemented in the MVS.
+:Reason: The MVS tool is a work in progress and this can still be addressed in the future.
+:Implications: The absence of such indicators might affect decision-making.
+
+Random excess energy distribution
+#################################
+:Limitation: There is random excess distribution between the feed-in sink and the excess sink when no feed-in-tariff is assumed in the system.
+:Reason: Since there is no feed-in-tariff to benefit from, the MVS randomly distributes the excess energy between the feed-in and excess sinks.
+:Implications: This distribution of excess energy changes when running several simulations for the same input files.
+
+Renewable energy share defintion
+################################
+:Limitation: The current renewable energy share depends on the share of renewable energy production assets directly feeding the load.
+:Reason: The MVS tool is a work in progress and this can still be addressed in the future.
+:Implications: This might result in different values when comparing them to other models.
+
+Energy carrier rating
+#####################
+:Limitation: The MVS assumes a usable potential/energy content rating for every energy carrier.
+:Reason: This is an approach that the MVS currently uses.
+:Implications: By weighing the energy carriers according to their energy content (Gasoline Gallon Equivalent (GGE)), the MVS might result in values that can't be directly assessed. Those ratings affect the calaculation of the levelized cost of the energy carriers, but also the minimum renewable energy share constraint.
+
+Energy shortage or grid interruption
+####################################
+:Limitation: The MVS assumes no shortage or grid interruption in the system.
+:Reason: 
+:Implications: Electricity shortages due to power cuts might happen in real life. The MVS currently omits this scenario which might implicate unreasonable results. If a system is self-sufficient but relies on grid-connected PV systems, the latter stop feeding the load if any power cuts occur and the battery storage systems might not be enough to serve the load (energy shortage).
+
+Generic cost definitions
+########################
+:Limitation: The MVS includes development costs, specific costs, O&M costs and disptach costs.
+:Reason: The MVS assumes those costs only for simplification.
+:Implications: This could lead to a rough estimation of the entire project costs.
+
+Unlinked one technical unit transformers
+########################################
+:Limitation: Some components should be modelled as separate transformers in the MVS while in real life they are one technical unit.
```

Comment:
That in itself is not a limitation yet. What does this do to the model that is wrong?